### ISSUE TITLE
license question: API boundary or infectious?

### ISSUE THREAD

**Issue by egonw** [Juris-M/citeproc-js#77]
I was wondering about the license (CPAL/AGPL) and particularly how the API boundary is being interpreted. Do you consider citeproc.js a library with API and that use of the library implies the AGPL/CPAL requirement (infection) for using tools? Or, alternatively, use of citeproc.js as a library does not require tools to use AGPL/CPAL license as well, but allow, for example, also using code to have other OSI-license, like MIT?

**Comment by fbennett** [comment]
Could you rephrase that? (Just because the two alternatives seem to mean the same thing, so I'm not sure I understand the specific concern.)

**Comment by egonw** [comment]
Sorry, my bad. Fixed now.

**Comment by fbennett** [comment]
Thanks, got it! I only mean for the chosen license (CPAL/AGPL) to apply to the code of citeproc-js itself. Another OSI license on calling code is no problem from where I sit.
